Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -341,6 +341,41 @@ public IUploadProgress GetProgress()
 
         #endregion
 
+        #region UploadSessionData
+        /// <summary>
+        /// <para>Event called when an UploadUri is created.</para>
+        /// <para>Not needed if the application program will not support resuming after a program restart.</para>
+        /// <para>Within the event, persist the UploadUri to storage.</para>
+        /// <para>It is strongly recommended that the full path filename (or other media identifier) is also stored so that it can be compared to the current open filename (media) upon restart.</para>
+        /// </summary>

```

Comment:
Please move most of this summary to remarks, just keep the first line (without the para) as the summary.
And for all other large summaries.